src/vmshepherd/http/rpc_api.py

```diff
@@ -77,7 +77,7 @@ def rpc_logger(func):
         vmshepherd = self.request.app.vmshepherd
         preset = vmshepherd.preset_manager.get_preset(preset)
         await preset.refresh_vms()
-        result_vms = {vm.id: {'ip': vm.ip[0], 'state': vm.state.value} for vm in preset.vms}
+        result_vms = {vm.id: {'ip': vm.ip[0], 'state': vm.state.value, 'created': vm.created} for vm in preset.vms}
```

Comment:
Are you sure Openstack returns dt as timestamp not as datetime object nor some preformatted string?

Reply:
In all our cases (AWS, openstack) it will be datetime object and after convert json to string it change to preformatted string.

https://github.com/DreamLab/VmShepherd/blob/master/src/vmshepherd/iaas/vm.py#L63